desktop/apps/personalize/client/onboarding.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+import App from 'desktop/apps/personalize/components/App'
+import { rehydrateClient } from 'desktop/components/react/utils/renderReactLayout'
+
+export const init = () => {
+  // Rehydrate data from Server
+  const bootstrapData = rehydrateClient(window.__BOOTSTRAP__)
```

Comment:
@kanaabe - I'm about 99% sure this is no longer needed now that we have stitch. Can you just try assigning `bootstrapData = window.__BOOTSTRAP__` and see what happens? I've been meaning to remove references to this.

Reply:
@damassi when we didn't have `rehydrateClient` we were losing our CSS and JS, but I'll give your suggestion a try.

Reply:
Oh interesting, if thats so l'll def take a look.

Reply:
@damassi yep that worked to remove rehydrateClient!